packages/client/src/components/FooterSection/FooterSection.component.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './FooterSection.style.css';
+import { Link } from "react-router-dom";
+import vectorFacebook from '../../../public/assets/vectors/vectorFacebook.svg';
+import vectorInstagram from '../../../public/assets/vectors/vectorInstagram.svg';
+import vectorLinkedin from '../../../public/assets/vectors/vectorLinkedin.svg';
+
+export const FooterSection = ({ size }) => {
+  return (
+    <footer className="footerSection">
+      <section className="footerContentSection">
```

Comment:
I don't understand why `section` is being used in this component. 

`section` is semantic in the same way as `p` or `h1` is, it describes the layout of a text. This feels more like just a `div` to group elements together.

I'm not saying you should use `div` for everything but in many cases `div` is the right choice. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
Just chiming in here 😅 This is a bit different from what I taught in class. You might have a point, but in my view `div` is to be avoided as it has no semantic meaning. It's a `div`ider, nothing more. Only use it if you have exhausted all other options. 

I think `section` isn't necessarily a bad choice here, while still being wildly generic (but at least less generic than `div`). Possibly `article` could be a better fit.

> The <section> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it.

Reply:
We changed section to an article and in some places we used div also :)

Reply:
Great 👍 Sorry if I caused confusion 😅 

After talking to Milton a bit more, I realized that I might need to update my knowledge on semantic HTML in webapps... Please take his feedback into consideration even if it might contradict what we discussed in classes. 

Ultimately, I think implementation details are up to you as a group to decide on – if they don't fundamentally break the site.

Reply:
Thank you for your quick response. We fixed it now 👍